### DISCUSSION TITLE
Git integration is completely broken for me

### DISCUSSION THREAD

**Discussion by ayush-gravityblogs** [GitbookIO/community#434] [3. Git Sync]
Today, I installed github integration

**Error 1:**
It showed installed successfully but the install button is still there.
Now when I click on the Install button again, I see this error:
![image](https://github.com/GitbookIO/community/assets/154821898/1237cc7a-9eec-4724-a4cc-fd6f339dc02a)

Error 2:
Even though Github integration is installed, I am unable to see any option for connecting my repo:
![image](https://github.com/GitbookIO/community/assets/154821898/853e23e9-a2a0-4fa8-bcf7-2b9c5e479cdd)

Error 3:
After sometime, It showed me "Connect github" button, I clicked, it asked me to login so I logged in, and that's it.
Nothing happened after that. Still shows the same "Connect github" button.

What's happening?

**Comment by aikoii** [comment]
@ayush-gravityblogs, thanks for your report! We'll investigate this further and keep you updated 🙏

**Reply by aikoii** [reply]
Hey there, this should be fixed 🙂

**Reply by geekyayush** [reply]
thank you so much.
sorry for late reply.
